src/Workspace.cpp

```diff
@@ -357,10 +362,7 @@ void WorkspaceFolder::setupWithConfiguration(const ClientConfiguration& configur
         platform = LSPPlatform::getPlatform(configuration, &fileResolver, this);
         fileResolver.platform = platform.get();
 
-        client->sendTrace("workspace: applying platform mutations on definitions");
-
-        platform->mutateRegisteredDefinitions(frontend.globals, definitionsFileMetadata);
-        platform->mutateRegisteredDefinitions(frontend.globalsForAutocomplete, definitionsFileMetadata);
+        registerTypes();
```

Comment:
Since this is in setupWithConfiguration, this is now going to re-register all the builtins whenever config changes, when it did not use to do that.

That is potentially a problem, can we make it only register once on startup like before?

Reply:
I think this is not an issue, see context: https://github.com/JohnnyMorganz/luau-lsp/blob/a1f9302db247bd6949c98c5d8fbc6e37bcd0ef22/src/Workspace.cpp#L356-L366

Reply:
Whoops, my bad! GitHub decided to hide those extra lines from me 🤦‍♂️